api/v1/models/UniversalTrackingItem.js

```diff
@@ -0,0 +1,17 @@
+var Model = require('./Model');
+
+var UniversalTrackingItem = Model.extend({
+    tableName: 'event_tracking',
```

Comment:
This doesn't follow our convention. See comment on migration. The name of the model should also _probably_ match the table name somewhat.